### PR TITLE
update conventions

### PR DESCRIPTION
**Pull request by lcnr**
Not sure about the exact wording yet.

### CODE REVIEW

**Comment by lcnr** [comment]
```
     ┌── appendix/background.md:145:3 ───

     │

 145 │ - [Compilers: Principles, Techniques, and Tools, 2nd Edition](https://www.amazon.com/dp/9332518661/ref=cm_sw_r_other_apa_1tUSBb5VHAVA1)

     │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Server responded with 503 Service Unavailable
```

File: src/conventions.md [hunk 1]
```diff
@@ -7,14 +7,18 @@ chapter covers [formatting](#formatting), [coding for correctness](#cc),
 
 # Formatting and the tidy script
 
-rustc is slowly moving towards the [Rust standard coding style][fmt];
-at the moment, however, it follows a rather more *chaotic* style.  We
-do have some mandatory formatting conventions, which are automatically
-enforced by a script we affectionately call the "tidy" script.  The
-tidy script runs automatically when you do `./x.py test` and can be run
+rustc is moving towards the [Rust standard coding style][fmt].
+This is enforced by the "tidy" scipt and can be mostly
```
Comment (spastorino) [inline src/conventions.md:11]:
```suggestion
This is enforced by the "tidy" script and can be mostly
```

File: src/conventions.md [hunk 2]
```diff
@@ -114,12 +118,9 @@ you introduce some code following one strategy, then change it
...
 
-**Only run rustfmt on new content.** One day, we might enforce formatting
-for the rust-lang/rust repo. Meanwhile, we prefer that rustfmt not be run
-on existing code as that will generate large diffs and will make git blame
-harder to sift through. However, running `rustfmt` on new content, e.g. a
-new file or a largely new part of a file is ok. Small formatting adjustments
-nearby code you are already changing for other purposes are also ok.
+**Format liberally.** While only the final commit of a PR must be correctly
+formatted, it is both easier to review and less noisy to format each commit
+individually using `./x.py fmt`.
```
Comment (mark-i-m) [inline src/conventions.md:123]: :+10000:

**Comment by mark-i-m** [comment]
@lcnr when you fix Santiago's comment, could you also rebase over master?

**Comment by lcnr** [comment]
Done. RFM I guess

**Pull request merged**